cli/util/util.go

```diff
@@ -899,3 +899,28 @@ func InstantiateFileFromTemplate(templatePath string, templateContent string,
 	}
 	return nil
 }
+
+// CollectAppList collects all the supposed applications from the passed directory,
+// considering the passed slice of exclude regexp items.
+func CollectAppList(baseDir string) ([]string, error) {
+	dirEnrties, err := os.ReadDir(baseDir)
+	if err != nil {
+		return nil, err
+	}
+
+	apps := make([]string, 0)
+	for _, entry := range dirEnrties {
+		dirItem := filepath.Join(baseDir, entry.Name())
+		if IsApp(dirItem) {
+			apps = append(apps, entry.Name())
+		} else {
+			log.Warnf("The application %s can't be packed: failed to access the source",
```

Comment:
We can use CollectAppList function only for packing, right?

Reply:
Fixed.